### PR TITLE
Update python dependencies

### PR DESCRIPTION
**Pull request by alles-klar**
Includes also some updates which are security relevant.

On behalf of DB Systel GmbH

---

**Note: DefectDojo is now on Python3 and Django 2.2.1 Please submit your pull requests to the 'dev' branch as the 'legacy-python2.7' branch is only for bug fixes. Any new features submitted to the legacy branch will be ignored and closed.**

When submitting a pull request, please make sure you have completed the following checklist:

- [x] Your code is flake8 compliant 
- [x] Your code is python 3.5 compliant
- [x] If this is a new feature and not a bug fix, you've included the proper documentation in the ReadTheDocs documentation folder. https://github.com/DefectDojo/Documentation/tree/master/docs or provide feature documentation in the PR.
- [x] Model changes must include the necessary migrations in the dojo/dd_migrations folder.
- [x] Add applicable tests to the unit tests.

### CODE REVIEW

**Comment by madchap** [comment]
I just overrode dependabot config via the UI since the config.yml seems to have issues to be taken into account.. PRs from the bot should come soon. I'd rather for us wait for these and work from there. OK with you?

**Comment by madchap** [comment]
Closing and re-opening to re-trigger Travis. It's really bad these days...

**Comment by madchap** [comment]
I see you've made some changes there in a couple .py files. Were they just because of deprecation warnings, or did it actually stop working altogether after you updated some of these deps?

**Comment by alles-klar** [comment]
There are some braking changes, so it stopped working without these changes.

I don't prefer to wait for dependabot and get a lot of PR. It is less work to update these dependencies on bulk at these point. For the future I agree to wait for dependabot. But at the moment there are to many updates we missed in the past.

**Comment by madchap** [comment]
I would agree yes. 

Just took a quick look at Travis, the errors that pop up in there seem legit. Could you please take a look?

Cheers.

File: requirements.txt [hunk 1]
```diff
@@ -1,49 +1,49 @@
...
+lxml==4.4.2
 Markdown==3.1.1
-mysqlclient==1.4.2
+mysqlclient==1.4.6
 pandas>=0.24.2
 pdfkit==0.6.1
-Pillow==6.0.0  # required by django-imagekit
-psycopg2-binary==2.8.2
-cryptography==2.7
+Pillow==7.0.0  # required by django-imagekit
```
Comment (madchap) [inline requirements.txt]: Probably would be more conservative and use 6.2.0?

Comment (alles-klar) [reply requirements.txt]: It is working, so why should't we use the latest version?

Comment (madchap) [reply requirements.txt]: If you're certain it's working, then OK :-)

**Comment by madchap** [comment]
Opening and closing to see if the Travis errors repeat themselves.

Otherwise @alles-klar, would you have time to take a look at them and find out what could be wrong?

**Comment by Maffooch** [comment]
API V1 was broken by something here. You can see the errors by looking at the logs of the failed travis tests.

**Comment by madchap** [comment]
I rebuilt my images and ran tests locally (`docker exec -ti madchap_ddojo_uwsgi_1 bash -c 'python3 manage.py test dojo.unittests -v2'`), and I don't get this error :-/

**Pull request merged**